codec-http2/src/main/java/io/netty/handler/codec/http2/Http2Stream.java

```diff
@@ -158,6 +158,18 @@
     Http2Stream parent();
 
     /**
+     * Returns {@code true} if this node is viable in its own right to be in the priority tree and {@code false}
+     * otherwise. For example if {@link #state()} returns {@link State#CLOSED} then this node is not considered viable.
+     * Once a {@link Http2Stream} becomes inviable it will not transition back to viable.
+     */
+    boolean isViable();
```

Comment:
When I choose this nebulous name I didn't use `closed` because I wasn't sure if other states would need to be included here.  I like `isPrioritizable()` because we are really trying to define a concept that says "if this node by itself belongs in the priority tree".  We also need something that says "do any of this node's descendants belong in the priority tree"...`hasPrioritizableDescendant()`?